packages/paste-core/components/chat-composer/src/ChatComposerContainer.tsx

```diff
@@ -0,0 +1,76 @@
+import type { BoxProps } from "@twilio-paste/box";
+import { Box, safelySpreadBoxProps } from "@twilio-paste/box";
+import { ThemeShape } from "@twilio-paste/theme";
```

Comment:
question: what is this?

Reply:
I used this as that was the way I saw Shadi do it for AI components. I can see that other areas of the code base just have the type to be BoxStyleProps. I could do it like this if it is the correct way:

```
const defaultStyles: BoxStyleProps = merge(BaseStyles.default, {
  color: "colorTextLinkDestructive",
  textAlign: "left",
  transition: "none",
  _hover: { color: "colorTextLinkDestructiveStrongest", textDecoration: "underline" },
  _active: { color: "colorTextLinkDestructiveStrongest", textDecoration: "underline" },
});
```

Reply:
I went with changing it to BoxStyleProps. It worked far better with types for the properties and is used elsewhere in the repo. I also updated the AI components where it was incorrectly used.